cScripts/functions/modules/fn_zenModule_createMedicalCrate.sqf

```diff
@@ -47,7 +47,7 @@ private _displayTextList =
         
         private _groundLevel = ASLToAGL _modulePos;
         private _medCrateContents = [_crateType] call FUNC(getMedicalCrate);
-        private _crate = ["ace_medicalSupplyCrate", _medCrateContents, _groundLevel] call FUNC(createCargoCrate); 
+        private _crate = ["ace_medicalSupplyCrate", _medCrateContents, _groundLevel] remoteExec [QFUNC(createCargoCrate),0,true]; 
```

Comment:
`_groundLevel = ASLToAGL _modulePos`
could just be `_modulePos` cause it's litterly where you drop the module.

Reply:
I put that line in because I was having issues with the crate spawning in the air. Unsure what was causing it but that is what fixed it.

Reply:
> Also we should take a look at the other crate modules then because they all use `0` for their execution scope as well

It depends on the crate if it have aceActions or addActions they are all local.